package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
+    "bulma": "^0.1.2",
```

Comment:
@jbailey4 @crodriguez1a Yea I'm good with that, cut a minor version update `0.2.0`